src/site/xdoc/checks/misc/translation.xml.template

```diff
@@ -93,17 +93,29 @@ messages.properties: Key 'ok' missing.
         </p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/translation/Example1.txt"/>
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/translation/Example1.java"/>
           <param name="type" value="config"/>
         </macro>
         <p id="Example1-code">
-          Example:
+          Example1:
+        </p>
+        <p id="Example1-messagesProperties">
+          Contents of messages.properties file:
+        </p>
+        <source>
+hello=Hello
```

Comment:
please review update https://github.com/checkstyle/checkstyle/pull/16141/files#diff-1a0e99bbb8140dd6852b3339803fbdbf9b106347f8a55ee9cc93f38f87995b7dR54

to see how to use "raw" type and put in web site content of whole file of properties.
content shoudl not be hardcoded in template, it should be loaded from property file.